src/style/globalStyle.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+html {
+    box-sizing: border-box;
+}
+*, ::after, ::before {
```

Comment:
usually this is as a first one as is affects all elements, then `html` and `body`